src/main/java/me/coley/recaf/mapping/TSrgMappings.java

```diff
@@ -0,0 +1,66 @@
+package me.coley.recaf.mapping;
+
+import me.coley.recaf.util.StringUtil;
+import me.coley.recaf.workspace.Workspace;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * TSRG mappings file implementation.
+ *
+ * @author Filip (1fxe)
+ */
+public class TSrgMappings extends FileMappings {
+    private static final String FAIL = "Invalid TSrg mappings, ";
```

Comment:
Does this really need to be a class level constant? And if so, why not have `"failed parsing line "` as a constant too? Seems inconsistent.

Reply:
I wrote the constant to be consistent with the other mapping files. For example https://github.com/Col-E/Recaf/blob/419925d996fe33d3caa94a864070af97c1a2d625/src/main/java/me/coley/recaf/mapping/SrgMappings.java#L19

https://github.com/Col-E/Recaf/blob/419925d996fe33d3caa94a864070af97c1a2d625/src/main/java/me/coley/recaf/mapping/TinyV1Mappings.java#L19

Reply:
Ah ok, looks good then